rbc/tests/test_omnisci_column_basic.py

```diff
@@ -223,14 +206,14 @@ def add_columns(x, y, alpha, m, r):
 def test_rowmul_return_two_columns(omnisci):
     if omnisci.has_cuda:
         pytest.skip('crashes CUDA enabled omniscidb server'
-                    ' [rbc issue 147]')
+                    ' [rbc issue 171]')
     omnisci.reset()
     # register an empty set of UDFs in order to avoid unregistering
     # UDFs created directly from LLVM IR strings when executing SQL
     # queries:
     omnisci.register()
 
-    @omnisci('int32(Column<double>, int64|sizer=RowMultiplier,'
+    @omnisci('int32(Column<double>, RowMultiplier,'
```

Comment:
RBC will automatically convert `RowMultiplier` to `int32|sizer=RowMultiplier`?

Reply:
No. There will be an exception raised, see
https://github.com/xnd-project/rbc/pull/170/files#diff-8e1cf1ddf04a5e6c92ef9bb819f2a10dR627